h/static/scripts/site.js

```diff
@@ -29,12 +29,13 @@ var controllers = {
   '.js-tooltip': TooltipController,
 };
 
-var doUpgrade = !window.envFlags || window.envFlags.get('js-capable');
-
-if (doUpgrade) {
-  upgradeElements(document.body, controllers);
-}
-
 if (window.envFlags) {
+  upgradeElements(document.body, controllers, {
```

Comment:
So you're no longer checking the `js-capable` flag? Doesn't this mean that you can no longer simulate no-js-capable mode using the query string?